vhd-format.opam

```diff
@@ -8,26 +8,16 @@ authors: [ "Dave Scott" "Jon Ludlam" ]
 homepage: "https://github.com/mirage/ocaml-vhd"
 bug-reports: "https://github.com/mirage/ocaml-vhd/issues"
 dev-repo: "git://github.com/mirage/ocaml-vhd"
-build: make
-remove: [[make "uninstall"]]
+build: [[ "jbuilder" "build" "-p" name "-j" jobs ]]
 depends: [
-  "ocamlfind"
-  "lwt" {>= "2.4.3"}
   "cstruct" {>= "1.9"}
-  "mirage-block"
-  "mirage-types-lwt" {>= "3.0.0"}
-  "ipaddr"
   "io-page"
-  "io-page-unix"
-  "uuidm"
-  "ounit"
   "rresult"
+  "uuidm"
   "ppx_cstruct" {build}
   "ppx_tools" {build}
```

Comment:
Hmm, I wonder whether we really need this dependency? 🤔 - ppx_cstruct itself depends on something called ppx_tools_versioned.

Reply:
What happens if you try to remove it?

Reply:
I'll try. The Travis build should fail if it's needed, unless it's a transitive dependency.